Script to check Sentiment/script.py

```diff
@@ -0,0 +1,92 @@
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
```

Comment:
wrong implementation. It's mentioned very clearly in the title also that you cannot use any external library for this: 

![image](https://user-images.githubusercontent.com/43691873/115178875-cd7bc280-a0ef-11eb-9675-555f2249d2cc.png)

Kind;y check the issue description again and then make changes to this PR

Reply:
@kaustubhgupta then is there any way to find the percentage of positive, negative and neutral.

Reply:
I think you need to create a separate corpus for positive, negative, and neutral cases. This is also mentioned in the issue description